build/image.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 SUSE
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit -o nounset -o pipefail
+
+git_root=$(git rev-parse --show-toplevel)
```

Comment:
Nit: quote.  Because I've been forever scarred by `C:\Program Files`.
```suggestion
git_root="$(git rev-parse --show-toplevel)"
```

Request for enhancement (in a separate PR): linting for this repo (probably via GitHub Actions or similar).

Reply:
I'll take the suggestion, but there's no harm here. We don't get word splitting on a variable declaration.
https://repl.it/repls/OldlaceFixedAmoeba